common/src/main/java/ch/admin/bag/covidcertificate/common/faq/FaqItem.kt

```diff
@@ -77,7 +77,7 @@ data class QuestionItem(
 
 		if (hasLink) {
 			binding.itemFaqQuestionLinkLabel.text = question.linkTitle
-			binding.itemFaqQuestionLink.setOnClickListener { onLinkClickListener?.onLinkClicked(question.linkUrl!!) }
+			binding.itemFaqQuestionLink.setOnClickListener { onLinkClickListener?.invoke(question.linkUrl!!) }
```

Comment:
I'd feel safer if it was written in a way such that the compiler could deduce from the `hasLink` check that `linkUrl` is nonnull.